src/components/autocomplete/autocomplete-theme.scss

```diff
@@ -13,6 +13,44 @@ md-autocomplete.md-THEME_NAME-theme {
       background: '{{background-600-0.3}}';
     }
   }
+  &.md-accent {
+    md-input-container.md-input-focused {
+      .md-input {
+        border-color: '{{accent-color}}';
+      }
+      label,
+      md-icon {
+        color: '{{accent-color}}';
+      }
+    }
+    md-progress-linear {
+      .md-container {
+        background-color: '{{accent-100}}';
+      }
+      .md-bar {
+        background-color: '{{accent-color}}';
+      }
+    }
+  }
+  &.md-warn {
+    md-input-container.md-input-focused {
+      .md-input {
+        border-color: '{{warn-A700}}';
+      }
+      label,
+      md-icon {
+        color: '{{warn-A700}}';
+      }
+    }
+    md-progress-linear {
+      .md-container {
+        background-color: '{{warn-100}}';
+      }
+      .md-bar {
+        background-color: '{{warn-color}}';
+      }
+    }
+  }
```

Comment:
If we are doing something like that, then we should create a mixin for it, so we don't have to repeat it three times.